std_msgs/String camname
std_msgs/Int8 id
exam/Pose spot
